cmd/sriov/main.go

```diff
@@ -270,16 +270,10 @@ func cmdDel(args *skel.CmdArgs) error {
 
 	sm := sriov.NewSriovManager()
 
-	/* ResetVFConfig resets a VF administratively. We must run ResetVFConfig
-	   before ReleaseVF because some drivers will error out if we try to
-	   reset netdev VF with trust off. So, reset VF MAC address via PF first.
-	*/
-	if err := sm.ResetVFConfig(netConf); err != nil {
-		return fmt.Errorf("cmdDel() error reseting VF: %q", err)
-	}
+	var netns ns.NetNS
 
 	if !netConf.DPDKMode {
-		netns, err := ns.GetNS(args.Netns)
+		netns, err = ns.GetNS(args.Netns)
```

Comment:
I think we should always do this not only for non dpdk devices.
if not we should be able to reproduce the issue on a vfio-pci device for example (maybe not using the ipam)
but for example if the driver is slow and we need to reset the vlan on the vf and that part takes a lot of time we maybe be able to reproduce the issue no?

Reply:
that's a good point. I updated the code so that if the net namespace doesn't exist, then Release and Reset are not called.
WDYT?